parachain/common/src/registry.rs

```diff
@@ -2,17 +2,24 @@
 use crate::types::AppID;
 
 pub enum AppName {
-    PolkaETH
+	PolkaETH,
+	PolkaERC20,
 }
 
 pub struct Entry {
-    pub symbol: AppName,
-    pub id: AppID,
+	pub symbol: AppName,
+	pub id: AppID,
 }
 
 pub static REGISTRY: &'static [Entry] = &[
```

Comment:
On Ethereum the Bank contract is one application with two distinct events/tags. If my understanding is correct and this is an application registry (as opposed to an event registry) then the relationship between applications is not 1:1. Does that introduce any potential issues for bi-directional communication between applications?

Reply:
I definitely still want a 1-1 correspondence between AppId's and the ethereum event tags (SendETH or SendERC20). I didn't manage to wire up that routing in this milestone though.